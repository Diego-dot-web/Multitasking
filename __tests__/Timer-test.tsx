import renderer from "react-test-renderer";
import Timer from "../app/(tabs)/Timer";

jest.mock("../context/ThemeContext");
jest.useFakeTimers();

it("renders Timer component", async () => {
  const tree = await renderer.create(<Timer />).toJSON();

  expect(tree).toMatchSnapshot();
});
