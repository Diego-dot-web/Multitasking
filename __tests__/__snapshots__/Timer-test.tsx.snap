// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Timer component 1`] = `
<RCTScrollView
  contentContainerStyle={
    {
      "padding": 16,
    }
  }
  style={
    [
      {
        "flex": 1,
      },
      {
        "backgroundColor": "#09090B",
      },
    ]
  }
>
  <View>
    <View
      style={
        {
          "marginBottom": 16,
        }
      }
    >
      <Text
        style={
          [
            {
              "fontSize": 24,
              "fontWeight": "bold",
              "marginBottom": 4,
            },
            {
              "color": "#FAFAFA",
            },
          ]
        }
      >
        Forja del Tiempo
      </Text>
      <Text
        style={
          [
            {
              "fontSize": 16,
            },
            {
              "color": "#71717A",
            },
          ]
        }
      >
        Utiliza la técnica Pomodoro para maximizar tu concentración y productividad.
      </Text>
    </View>
    <View
      style={
        {
          "gap": 16,
        }
      }
    >
      <View
        style={
          [
            {
              "borderRadius": 8,
              "borderWidth": 1,
              "elevation": 2,
              "padding": 16,
              "shadowColor": "#000",
              "shadowOffset": {
                "height": 2,
                "width": 0,
              },
              "shadowOpacity": 0.1,
              "shadowRadius": 4,
            },
            {
              "backgroundColor": "#18181B",
              "borderColor": "#27272A",
            },
            {
              "padding": 16,
            },
          ]
        }
      >
        <View
          style={
            {
              "alignItems": "flex-start",
              "flexDirection": "row",
              "justifyContent": "space-between",
              "marginBottom": 16,
            }
          }
        >
          <View>
            <View
              style={
                {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "gap": 8,
                  "marginBottom": 4,
                }
              }
            >
              <RNSVGSvgView
                align="xMidYMid"
                bbHeight={20}
                bbWidth={20}
                fill="none"
                focusable={false}
                height={20}
                meetOrSlice={0}
                minX={0}
                minY={0}
                stroke="#9333EA"
                strokeLinecap="round"
                strokeLinejoin="round"
                strokeWidth="2"
                style={
                  [
                    {
                      "backgroundColor": "transparent",
                      "borderWidth": 0,
                    },
                    {
                      "flex": 0,
                      "height": 20,
                      "width": 20,
                    },
                  ]
                }
                vbHeight={24}
                vbWidth={24}
                width={20}
              >
                <RNSVGGroup
                  fill={null}
                  propList={
                    [
                      "fill",
                      "stroke",
                      "strokeWidth",
                      "strokeLinecap",
                      "strokeLinejoin",
                    ]
                  }
                  stroke={
                    {
                      "payload": 4287837162,
                      "type": 0,
                    }
                  }
                  strokeLinecap={1}
                  strokeLinejoin={1}
                  strokeWidth="2"
                >
                  <RNSVGCircle
                    cx="12"
                    cy="12"
                    fill={
                      {
                        "payload": 4278190080,
                        "type": 0,
                      }
                    }
                    r="10"
                  />
                  <RNSVGPath
                    d="M10 8l6 4-6 4V8z"
                    fill={
                      {
                        "payload": 4278190080,
                        "type": 0,
                      }
                    }
                  />
                </RNSVGGroup>
              </RNSVGSvgView>
              <Text
                style={
                  [
                    {
                      "fontSize": 18,
                      "fontWeight": "bold",
                    },
                    {
                      "color": "#9333EA",
                    },
                  ]
                }
              >
                Tiempo de Enfoque
              </Text>
            </View>
            <Text
              style={
                [
                  {
                    "fontSize": 14,
                  },
                  {
                    "color": "#71717A",
                  },
                ]
              }
            >
              Concéntrate en tu tarea actual
            </Text>
          </View>
          <View
            style={
              {
                "flexDirection": "row",
                "gap": 8,
              }
            }
          >
            <View
              accessibilityState={
                {
                  "busy": undefined,
                  "checked": undefined,
                  "disabled": undefined,
                  "expanded": undefined,
                  "selected": undefined,
                }
              }
              accessibilityValue={
                {
                  "max": undefined,
                  "min": undefined,
                  "now": undefined,
                  "text": undefined,
                }
              }
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                {
                  "opacity": 1,
                  "padding": 8,
                }
              }
            >
              <RNSVGSvgView
                align="xMidYMid"
                bbHeight={16}
                bbWidth={16}
                fill="none"
                focusable={false}
                height={16}
                meetOrSlice={0}
                minX={0}
                minY={0}
                stroke="#71717A"
                strokeLinecap="round"
                strokeLinejoin="round"
                strokeWidth="2"
                style={
                  [
                    {
                      "backgroundColor": "transparent",
                      "borderWidth": 0,
                    },
                    {
                      "flex": 0,
                      "height": 16,
                      "width": 16,
                    },
                  ]
                }
                vbHeight={24}
                vbWidth={24}
                width={16}
              >
                <RNSVGGroup
                  fill={null}
                  propList={
                    [
                      "fill",
                      "stroke",
                      "strokeWidth",
                      "strokeLinecap",
                      "strokeLinejoin",
                    ]
                  }
                  stroke={
                    {
                      "payload": 4285624698,
                      "type": 0,
                    }
                  }
                  strokeLinecap={1}
                  strokeLinejoin={1}
                  strokeWidth="2"
                >
                  <RNSVGPath
                    d="M18 8A6 6 0 0 0 6 8c0 7-3 9-3 9h18s-3-2-3-9"
                    fill={
                      {
                        "payload": 4278190080,
                        "type": 0,
                      }
                    }
                  />
                  <RNSVGPath
                    d="M13.73 21a2 2 0 0 1-3.46 0"
                    fill={
                      {
                        "payload": 4278190080,
                        "type": 0,
                      }
                    }
                  />
                </RNSVGGroup>
              </RNSVGSvgView>
            </View>
            <View
              accessibilityState={
                {
                  "busy": undefined,
                  "checked": undefined,
                  "disabled": undefined,
                  "expanded": undefined,
                  "selected": undefined,
                }
              }
              accessibilityValue={
                {
                  "max": undefined,
                  "min": undefined,
                  "now": undefined,
                  "text": undefined,
                }
              }
              accessible={true}
              collapsable={false}
              focusable={false}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                {
                  "opacity": 1,
                  "padding": 8,
                }
              }
            >
              <RNSVGSvgView
                align="xMidYMid"
                bbHeight={16}
                bbWidth={16}
                fill="none"
                focusable={false}
                height={16}
                meetOrSlice={0}
                minX={0}
                minY={0}
                stroke="#71717A"
                strokeLinecap="round"
                strokeLinejoin="round"
                strokeWidth="2"
                style={
                  [
                    {
                      "backgroundColor": "transparent",
                      "borderWidth": 0,
                    },
                    {
                      "flex": 0,
                      "height": 16,
                      "width": 16,
                    },
                  ]
                }
                vbHeight={24}
                vbWidth={24}
                width={16}
              >
                <RNSVGGroup
                  fill={null}
                  propList={
                    [
                      "fill",
                      "stroke",
                      "strokeWidth",
                      "strokeLinecap",
                      "strokeLinejoin",
                    ]
                  }
                  stroke={
                    {
                      "payload": 4285624698,
                      "type": 0,
                    }
                  }
                  strokeLinecap={1}
                  strokeLinejoin={1}
                  strokeWidth="2"
                >
                  <RNSVGCircle
                    cx="12"
                    cy="12"
                    fill={
                      {
                        "payload": 4278190080,
                        "type": 0,
                      }
                    }
                    r="3"
                  />
                  <RNSVGPath
                    d="M19.4 15a1.65 1.65 0 0 0 .33 1.82l.06.06a2 2 0 0 1 0 2.83 2 2 0 0 1-2.83 0l-.06-.06a1.65 1.65 0 0 0-1.82-.33 1.65 1.65 0 0 0-1 1.51V21a2 2 0 0 1-2 2 2 2 0 0 1-2-2v-.09A1.65 1.65 0 0 0 9 19.4a1.65 1.65 0 0 0-1.82.33l-.06.06a2 2 0 0 1-2.83 0 2 2 0 0 1 0-2.83l.06-.06a1.65 1.65 0 0 0 .33-1.82 1.65 1.65 0 0 0-1.51-1H3a2 2 0 0 1-2-2 2 2 0 0 1 2-2h.09A1.65 1.65 0 0 0 4.6 9a1.65 1.65 0 0 0-.33-1.82l-.06-.06a2 2 0 0 1 0-2.83 2 2 0 0 1 2.83 0l.06.06a1.65 1.65 0 0 0 1.82.33H9a1.65 1.65 0 0 0 1-1.51V3a2 2 0 0 1 2-2 2 2 0 0 1 2 2v.09a1.65 1.65 0 0 0 1 1.51 1.65 1.65 0 0 0 1.82-.33l.06-.06a2 2 0 0 1 2.83 0 2 2 0 0 1 0 2.83l-.06.06a1.65 1.65 0 0 0-.33 1.82V9a1.65 1.65 0 0 0 1.51 1H21a2 2 0 0 1 2 2 2 2 0 0 1-2 2h-.09a1.65 1.65 0 0 0-1.51 1z"
                    fill={
                      {
                        "payload": 4278190080,
                        "type": 0,
                      }
                    }
                  />
                </RNSVGGroup>
              </RNSVGSvgView>
            </View>
          </View>
        </View>
        <View
          style={
            {
              "alignItems": "center",
              "marginVertical": 16,
            }
          }
        >
          <View
            style={
              {
                "alignItems": "center",
                "height": 180,
                "justifyContent": "center",
                "width": 180,
              }
            }
          >
            <RNSVGSvgView
              bbHeight={180}
              bbWidth={180}
              focusable={false}
              height={180}
              style={
                [
                  {
                    "backgroundColor": "transparent",
                    "borderWidth": 0,
                  },
                  {
                    "flex": 0,
                    "height": 180,
                    "width": 180,
                  },
                ]
              }
              width={180}
            >
              <RNSVGGroup
                fill={
                  {
                    "payload": 4278190080,
                    "type": 0,
                  }
                }
              >
                <RNSVGCircle
                  cx={90}
                  cy={90}
                  fill={
                    {
                      "payload": 0,
                      "type": 0,
                    }
                  }
                  propList={
                    [
                      "fill",
                      "stroke",
                      "strokeWidth",
                    ]
                  }
                  r={70}
                  stroke={
                    {
                      "payload": 4280756010,
                      "type": 0,
                    }
                  }
                  strokeWidth={8}
                />
                <RNSVGCircle
                  cx={90}
                  cy={90}
                  fill={
                    {
                      "payload": 0,
                      "type": 0,
                    }
                  }
                  propList={
                    [
                      "fill",
                      "stroke",
                      "strokeWidth",
                      "strokeDasharray",
                      "strokeDashoffset",
                      "strokeLinecap",
                    ]
                  }
                  r={70}
                  stroke={
                    {
                      "payload": 4287837162,
                      "type": 0,
                    }
                  }
                  strokeDasharray={
                    [
                      439.822971502571,
                      439.822971502571,
                    ]
                  }
                  strokeDashoffset={439.822971502571}
                  strokeLinecap={1}
                  strokeWidth={8}
                />
              </RNSVGGroup>
            </RNSVGSvgView>
            <Text
              style={
                [
                  {
                    "fontSize": 36,
                    "fontVariant": [
                      "tabular-nums",
                    ],
                    "fontWeight": "bold",
                    "position": "absolute",
                  },
                  {
                    "color": "#FAFAFA",
                  },
                ]
              }
            >
              25:00
            </Text>
          </View>
        </View>
        <View
          style={
            {
              "flexDirection": "row",
              "gap": 12,
              "justifyContent": "center",
              "marginBottom": 16,
            }
          }
        >
          <View
            accessibilityState={
              {
                "busy": undefined,
                "checked": undefined,
                "disabled": undefined,
                "expanded": undefined,
                "selected": undefined,
              }
            }
            accessibilityValue={
              {
                "max": undefined,
                "min": undefined,
                "now": undefined,
                "text": undefined,
              }
            }
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "alignItems": "center",
                "backgroundColor": "#9333EA",
                "borderRadius": 8,
                "flexDirection": "row",
                "gap": 8,
                "justifyContent": "center",
                "opacity": 1,
                "paddingHorizontal": 24,
                "paddingVertical": 12,
              }
            }
          >
            <RNSVGSvgView
              align="xMidYMid"
              bbHeight={20}
              bbWidth={20}
              fill="none"
              focusable={false}
              height={20}
              meetOrSlice={0}
              minX={0}
              minY={0}
              stroke="#fff"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="2"
              style={
                [
                  {
                    "backgroundColor": "transparent",
                    "borderWidth": 0,
                  },
                  {
                    "flex": 0,
                    "height": 20,
                    "width": 20,
                  },
                ]
              }
              vbHeight={24}
              vbWidth={24}
              width={20}
            >
              <RNSVGGroup
                fill={null}
                propList={
                  [
                    "fill",
                    "stroke",
                    "strokeWidth",
                    "strokeLinecap",
                    "strokeLinejoin",
                  ]
                }
                stroke={
                  {
                    "payload": 4294967295,
                    "type": 0,
                  }
                }
                strokeLinecap={1}
                strokeLinejoin={1}
                strokeWidth="2"
              >
                <RNSVGCircle
                  cx="12"
                  cy="12"
                  fill={
                    {
                      "payload": 4278190080,
                      "type": 0,
                    }
                  }
                  r="10"
                />
                <RNSVGPath
                  d="M10 8l6 4-6 4V8z"
                  fill={
                    {
                      "payload": 4278190080,
                      "type": 0,
                    }
                  }
                />
              </RNSVGGroup>
            </RNSVGSvgView>
            <Text
              style={
                {
                  "color": "#fff",
                  "fontSize": 16,
                  "fontWeight": "600",
                }
              }
            >
              Iniciar
            </Text>
          </View>
          <View
            accessibilityState={
              {
                "busy": undefined,
                "checked": undefined,
                "disabled": undefined,
                "expanded": undefined,
                "selected": undefined,
              }
            }
            accessibilityValue={
              {
                "max": undefined,
                "min": undefined,
                "now": undefined,
                "text": undefined,
              }
            }
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "alignItems": "center",
                "borderColor": "#27272A",
                "borderRadius": 8,
                "borderWidth": 1,
                "flexDirection": "row",
                "gap": 8,
                "justifyContent": "center",
                "opacity": 1,
                "paddingHorizontal": 24,
                "paddingVertical": 12,
              }
            }
          >
            <RNSVGSvgView
              align="xMidYMid"
              bbHeight={20}
              bbWidth={20}
              fill="none"
              focusable={false}
              height={20}
              meetOrSlice={0}
              minX={0}
              minY={0}
              stroke="#FAFAFA"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="2"
              style={
                [
                  {
                    "backgroundColor": "transparent",
                    "borderWidth": 0,
                  },
                  {
                    "flex": 0,
                    "height": 20,
                    "width": 20,
                  },
                ]
              }
              vbHeight={24}
              vbWidth={24}
              width={20}
            >
              <RNSVGGroup
                fill={null}
                propList={
                  [
                    "fill",
                    "stroke",
                    "strokeWidth",
                    "strokeLinecap",
                    "strokeLinejoin",
                  ]
                }
                stroke={
                  {
                    "payload": 4294638330,
                    "type": 0,
                  }
                }
                strokeLinecap={1}
                strokeLinejoin={1}
                strokeWidth="2"
              >
                <RNSVGPath
                  d="M1 4v6h6"
                  fill={
                    {
                      "payload": 4278190080,
                      "type": 0,
                    }
                  }
                />
                <RNSVGPath
                  d="M3.51 15a9 9 0 1 0 2.13-9.36L1 10"
                  fill={
                    {
                      "payload": 4278190080,
                      "type": 0,
                    }
                  }
                />
              </RNSVGGroup>
            </RNSVGSvgView>
            <Text
              style={
                [
                  {
                    "fontSize": 16,
                    "fontWeight": "600",
                  },
                  {
                    "color": "#FAFAFA",
                  },
                ]
              }
            >
              Reiniciar
            </Text>
          </View>
        </View>
        <View
          style={
            {
              "flexDirection": "row",
              "gap": 8,
              "justifyContent": "space-between",
            }
          }
        >
          <View
            accessibilityState={
              {
                "busy": undefined,
                "checked": undefined,
                "disabled": undefined,
                "expanded": undefined,
                "selected": undefined,
              }
            }
            accessibilityValue={
              {
                "max": undefined,
                "min": undefined,
                "now": undefined,
                "text": undefined,
              }
            }
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "alignItems": "center",
                "backgroundColor": "#9333EA",
                "borderRadius": 8,
                "borderWidth": 1,
                "flex": 1,
                "justifyContent": "center",
                "opacity": 1,
                "paddingVertical": 10,
              }
            }
          >
            <Text
              style={
                [
                  {
                    "fontWeight": "500",
                  },
                  {
                    "color": "#fff",
                  },
                ]
              }
            >
              Enfoque
            </Text>
          </View>
          <View
            accessibilityState={
              {
                "busy": undefined,
                "checked": undefined,
                "disabled": undefined,
                "expanded": undefined,
                "selected": undefined,
              }
            }
            accessibilityValue={
              {
                "max": undefined,
                "min": undefined,
                "now": undefined,
                "text": undefined,
              }
            }
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "alignItems": "center",
                "borderColor": "#27272A",
                "borderRadius": 8,
                "borderWidth": 1,
                "flex": 1,
                "justifyContent": "center",
                "opacity": 1,
                "paddingVertical": 10,
              }
            }
          >
            <Text
              style={
                [
                  {
                    "fontWeight": "500",
                  },
                  {
                    "color": "#22C55E",
                  },
                ]
              }
            >
              Descanso Corto
            </Text>
          </View>
          <View
            accessibilityState={
              {
                "busy": undefined,
                "checked": undefined,
                "disabled": undefined,
                "expanded": undefined,
                "selected": undefined,
              }
            }
            accessibilityValue={
              {
                "max": undefined,
                "min": undefined,
                "now": undefined,
                "text": undefined,
              }
            }
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              {
                "alignItems": "center",
                "borderColor": "#27272A",
                "borderRadius": 8,
                "borderWidth": 1,
                "flex": 1,
                "justifyContent": "center",
                "opacity": 1,
                "paddingVertical": 10,
              }
            }
          >
            <Text
              style={
                [
                  {
                    "fontWeight": "500",
                  },
                  {
                    "color": "#9333EA",
                  },
                ]
              }
            >
              Descanso Largo
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          [
            {
              "borderRadius": 8,
              "borderWidth": 1,
              "elevation": 2,
              "padding": 16,
              "shadowColor": "#000",
              "shadowOffset": {
                "height": 2,
                "width": 0,
              },
              "shadowOpacity": 0.1,
              "shadowRadius": 4,
            },
            {
              "backgroundColor": "#18181B",
              "borderColor": "#27272A",
            },
            {
              "padding": 16,
            },
          ]
        }
      >
        <Text
          style={
            [
              {
                "fontSize": 18,
                "fontWeight": "bold",
              },
              {
                "color": "#FAFAFA",
              },
            ]
          }
        >
          Estadísticas
        </Text>
        <Text
          style={
            [
              {
                "fontSize": 14,
                "marginBottom": 16,
              },
              {
                "color": "#71717A",
              },
            ]
          }
        >
          Tu progreso de hoy
        </Text>
        <View
          style={
            {
              "marginBottom": 16,
            }
          }
        >
          <View
            style={
              {
                "flexDirection": "row",
                "justifyContent": "space-between",
                "marginBottom": 8,
              }
            }
          >
            <Text
              style={
                [
                  {
                    "fontSize": 14,
                    "fontWeight": "500",
                  },
                  {
                    "color": "#FAFAFA",
                  },
                ]
              }
            >
              Sesiones completadas
            </Text>
            <Text
              style={
                [
                  {
                    "fontSize": 14,
                    "fontWeight": "500",
                  },
                  {
                    "color": "#FAFAFA",
                  },
                ]
              }
            >
              0
            </Text>
          </View>
          <View
            style={
              [
                {
                  "backgroundColor": "#E2E8F0",
                  "borderRadius": 3,
                  "height": 6,
                  "overflow": "hidden",
                },
                {
                  "marginBottom": 8,
                },
              ]
            }
          >
            <View
              style={
                [
                  {
                    "borderRadius": 3,
                    "height": "100%",
                  },
                  {
                    "backgroundColor": "#9333EA",
                    "width": "0%",
                  },
                ]
              }
            />
          </View>
          <Text
            style={
              [
                {
                  "fontSize": 12,
                },
                {
                  "color": "#71717A",
                },
              ]
            }
          >
            Objetivo: 8 sesiones diarias
          </Text>
        </View>
        <View
          style={
            {
              "marginBottom": 16,
            }
          }
        >
          <Text
            style={
              [
                {
                  "fontSize": 14,
                  "fontWeight": "500",
                  "marginBottom": 8,
                },
                {
                  "color": "#FAFAFA",
                },
              ]
            }
          >
            Tiempo personalizado
          </Text>
          <View
            style={
              {
                "flexDirection": "row",
                "gap": 8,
              }
            }
          >
            <TextInput
              keyboardType="number-pad"
              maxLength={3}
              onChangeText={[Function]}
              style={
                [
                  {
                    "borderRadius": 4,
                    "borderWidth": 1,
                    "flex": 1,
                    "height": 44,
                    "paddingHorizontal": 12,
                    "textAlign": "center",
                  },
                  {
                    "backgroundColor": "#18181B",
                    "borderColor": "#27272A",
                    "color": "#FAFAFA",
                  },
                ]
              }
              value="25"
            />
            <View
              accessibilityState={
                {
                  "busy": undefined,
                  "checked": undefined,
                  "disabled": undefined,
                  "expanded": undefined,
                  "selected": undefined,
                }
              }
              accessibilityValue={
                {
                  "max": undefined,
                  "min": undefined,
                  "now": undefined,
                  "text": undefined,
                }
              }
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                {
                  "alignItems": "center",
                  "backgroundColor": "#71717A",
                  "borderRadius": 4,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingHorizontal": 16,
                }
              }
            >
              <Text
                style={
                  {
                    "color": "#fff",
                    "fontWeight": "500",
                  }
                }
              >
                Establecer
              </Text>
            </View>
          </View>
        </View>
        <View
          style={
            [
              {
                "borderRadius": 8,
                "padding": 12,
              },
              {
                "backgroundColor": "#71717A20",
              },
            ]
          }
        >
          <Text
            style={
              [
                {
                  "fontSize": 14,
                  "fontWeight": "500",
                  "marginBottom": 8,
                },
                {
                  "color": "#FAFAFA",
                },
              ]
            }
          >
            Consejos para la productividad
          </Text>
          <View
            style={
              {
                "alignItems": "flex-start",
                "flexDirection": "row",
                "marginBottom": 4,
              }
            }
          >
            <View
              style={
                [
                  {
                    "borderRadius": 3,
                    "height": 6,
                    "marginRight": 8,
                    "marginTop": 6,
                    "width": 6,
                  },
                  {
                    "backgroundColor": "#9333EA",
                  },
                ]
              }
            />
            <Text
              style={
                [
                  {
                    "flex": 1,
                    "fontSize": 12,
                  },
                  {
                    "color": "#71717A",
                  },
                ]
              }
            >
              Trabaja en bloques de 25 minutos con descansos cortos
            </Text>
          </View>
          <View
            style={
              {
                "alignItems": "flex-start",
                "flexDirection": "row",
                "marginBottom": 4,
              }
            }
          >
            <View
              style={
                [
                  {
                    "borderRadius": 3,
                    "height": 6,
                    "marginRight": 8,
                    "marginTop": 6,
                    "width": 6,
                  },
                  {
                    "backgroundColor": "#9333EA",
                  },
                ]
              }
            />
            <Text
              style={
                [
                  {
                    "flex": 1,
                    "fontSize": 12,
                  },
                  {
                    "color": "#71717A",
                  },
                ]
              }
            >
              Después de 4 bloques, toma un descanso largo
            </Text>
          </View>
        </View>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
